util/promote_staging_data

```diff
@@ -54,14 +55,17 @@ ALL_TEAMS=(cohort team-hafler team-hardy team-jakobsson team-lee team-scherzer t
 
 # Default to dry run if promotion is not selected
 DRY_RUN_ARG="-n"
+# Default to dev staging buckets
+STAGING_BUCKET_TYPE='dev'
```

Comment:
Maybe just for safety this could be `uat`? s.t. we don't accidentally promote an embargoed team that doesn't have any data in uat straight from dev -> prod